<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
    <line>
      <![CDATA[Homepage Verified]]>
    </line>
  </reporter-output>
  <suite name="Default suite" duration-ms="11683" started-at="2019-01-20T23:00:25Z" finished-at="2019-01-20T23:00:36Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="11683" started-at="2019-01-20T23:00:25Z" finished-at="2019-01-20T23:00:36Z">
      <class name="com.nike.allpages.test.RemoveTest">
        <test-method status="PASS" signature="openBrowser()[pri:0, instance:com.nike.allpages.test.RemoveTest@5442a311]" name="openBrowser" is-config="true" duration-ms="9254" started-at="2019-01-20T23:00:25Z" finished-at="2019-01-20T23:00:34Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- openBrowser -->
        <test-method status="FAIL" signature="homePageTest1()[pri:0, instance:com.nike.allpages.test.RemoveTest@5442a311]" name="homePageTest1" duration-ms="1863" started-at="2019-01-20T23:00:34Z" finished-at="2019-01-20T23:00:36Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertTrue(Assert.java:44)
at org.testng.Assert.assertTrue(Assert.java:54)
at com.nike.all_pages.HomePage.homePage_LogoCheck(HomePage.java:102)
at com.nike.allpages.test.RemoveTest.homePageTest1(RemoveTest.java:35)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
            <line>
              <![CDATA[Homepage Verified]]>
            </line>
          </reporter-output>
        </test-method> <!-- homePageTest1 -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.nike.allpages.test.RemoveTest@5442a311]" name="tearDown" is-config="true" duration-ms="0" started-at="2019-01-20T23:00:36Z" finished-at="2019-01-20T23:00:36Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.nike.allpages.test.RemoveTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
